spec/README.md

```diff
@@ -138,6 +139,15 @@ Type: `String`
 
 The associated language written in the native language.
 
+###### metadata.official
+
+Optional
+Type: `Array`
+
+An array of territories where the given language is officially spoken. This entry is automatically added by the build script, with information obtained directly from the [languageData](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/languageData.json) and [territoryInfo](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json) provided by the unicode-CLDR database.
+
+Any entries added to the language `src` file will be included with the appended territory list.
```

Comment:
What do you mean here? Do you mean that you can overwrite the build output by manually adding territories?